components/eam/src/physics/cam/shoc_intr.F90

```diff
@@ -389,6 +397,33 @@ subroutine shoc_init_e3sm(pbuf2d, dp1_in)
        edsclr_dim = edsclr_dim-1
     endif 
 
+    !SHOC needs all its water constituents in terms of "dry" mmr
+    !but the state vector has all its scalars in terms of "wet" mmr
+    !identify scalar which need conversion between wet and dry mmrs
+    do icnst = 1, pcnst
+       lq_dry_wet_cnvr(icnst) = .false. ! lets assume .false. by default
+       do idw = 1, size(dry_wet_scalars)
+          if(trim(adjustl(stateq_names(icnst))) == trim(adjustl(dry_wet_scalars(idw))) )then
```

Comment:
@singhbalwinder , where is this variable defined?  `stateq_names`.  I don't see it in any modules or being declared.